cypress/integration/inventory/actions-menu.spec.js

```diff
@@ -0,0 +1,23 @@
+import TopMenu from '../../support/fragments/topMenu';
+import Actions from '../../support/fragments/inventory/actions';
+
+describe('ui-inventory: actions', () => {
+  beforeEach('navigates to Actions', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(TopMenu.inventoryPath);
+  });
+
+  it('C196752 verifies action menu options before any search is conducted', () => {
+    const optionsShouldDisabled = [
```

Comment:
array can be located into Inventory with related getter

Reply:
I think it is bad idea, because state of options can be changed easly. Do I need safe a lot of arrays in Inventory with different states? It looks like excess work without profit

Reply:
discussed, current state is ok